apps/mull-ui-e2e/src/integration/create-event.spec.ts

```diff
@@ -1,4 +1,5 @@
 /// <reference types="Cypress" />
```

Comment:
Actually I just remembered, didn't we agree that we would make e2e tests, one for each user story?

Ideally, we should find a way to split this file into two: `US-1.1.spec.tsx` and `US-1.2.spec.tsx`.

However, I know that 1.2 is more of a supporting US for 1.1, and that their flow is quite intertwined, so I suggest that **at least** you indicate in this file that it covers both user stories. Both by renaming it to something like `US-1.1.US-1.2.spec.tsx` and changing the `describe` statement on line 10.

Reply:
I remember it was brought, don't remember the resolution we agreed on. The e2e for `US-1.2` is tightly coupled in at least one of the e2e tests for `US-1.1`. I will change the file name and `describe` statement for now we can discuss with team what we will do going forward.

Reply:
True